packages/modular-scripts/react-scripts/config/paths.js

```diff
@@ -46,7 +46,9 @@ const resolveModular = (relativePath) =>
 const publicUrlOrPath = getPublicUrlOrPath(
   process.env.NODE_ENV === 'development',
   require(resolveApp('package.json')).homepage,
-  process.env.PUBLIC_URL,
+  process.env.INTERNAL_PUBLIC_URL === ''
```

Comment:
letting getConfig return undefined would have been messy, but treating '' (empty string) as configuration not set might not be ideal either, open to suggestions

Reply:
Not sure what you mean here. Why is this value not consumed via `getConfig`?